site/source/user-manual/flashing.rst

```diff
@@ -0,0 +1,27 @@
+Getting EmbassyOS
+-----------------
+
+After building your device, you need a copy of EmbassyOS.
+
+* Visit `images.start9.com <https://images.start9.com>`_ and enter the product key ``xxxxxxxx`` (all lower case) to download an image of EmbassyOS.
+
+Installing EmbassyOS
+--------------------
+
+Whether you download EmbassyOS from our image server or build it yourself, you'll need to flash it onto a microSD card.
```

Comment:
replace: "from our image server"
with: "as a binary from github or our website"

Reply:
I don't see this change